data/common.yaml

```diff
@@ -1 +1,2 @@
 ---
+{} # <-- placeholder, prevents `file does not contain a valid yaml hash` warning
```

Comment:
Why not just remove the data directory and the hiera.yaml as well?  This looks like cruft from the module skeleton.